### PR TITLE
storage: Allow multiple alerts on details page

### PR DESCRIPTION
**Pull request by mvollmer**
And put them each in their own StackItem and Card, for proper padding and borders.

Before:

image

After:

image

### CODE REVIEW

File: pkg/storaged/vdo-details.jsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ export class VDODetails extends React.Component {
                         _("The creation of this VDO device did not finish and the device can't be used.")
                     </>} />
             );
-            return <StdDetailsLayout client={this.props.client} alert={broken} />;
+            return <StdDetailsLayout client={this.props.client} alerts={[broken]} />;
```
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:108]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19080-20230706-133236-58540325-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/vdo-details.jsx [hunk 2]
```diff
@@ -341,6 +341,6 @@ export class VDODetails extends React.Component {
 
         const content = <Block client={client} block={block} allow_partitions={false} />;
 
-        return <StdDetailsLayout client={this.props.client} alert={alert} header={header} content={content} />;
+        return <StdDetailsLayout client={this.props.client} alerts={[alert]} header={header} content={content} />;
```
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:344]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19080-20230706-133236-58540325-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Review by martinpitt — APPROVED** [review]
LGTM, but (1) I think you posted the same screenshot twice, there's no difference. And (2) @garrett still has a review finger on this, so I won't land it yet.

**Comment by mvollmer** [comment]
> but (1) I think you posted the same screenshot twice

No, there is a gap now between the alert and the card under it.

File: pkg/storaged/vdo-details.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:108]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19080-20230808-072541-6eff2589-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/vdo-details.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:344]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19080-20230808-072541-6eff2589-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Pull request merged**